docs/tutorials/ionq/getting_started.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<ul><li>January 2021?</li><li>Make a new line and start with <code>Note: </code>  to create a notice panel for the development version message to stand out. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done but you mean also a "&gt;"?  Or is there another way to do this in markdown</p>